build.psm1

```diff
@@ -169,12 +169,12 @@ function Start-PSBuild {
 
     # set output options
     $OptionsArguments = @{
-        Publish=$Publish
-        Output=$Output
-        FullCLR=$FullCLR
-        Runtime=$Runtime
-        Configuration=$Configuration
-        Verbose=$true
+        Publish=$Publish;
```

Comment:
Ah I was updating this for consistency with the other hashtables that do use `;`.